test/service/src/lib.rs

```diff
@@ -740,14 +753,15 @@ pub fn node_config(
 		state_pruning: Some(PruningMode::ArchiveAll),
 		blocks_pruning: BlocksPruning::KeepAll,
 		chain_spec: spec,
-		wasm_method: WasmExecutionMethod::Interpreted,
-		// NOTE: we enforce the use of the native runtime to make the errors more debuggable
+		wasm_method: WasmExecutionMethod::Compiled {
+			instantiation_strategy: WasmtimeInstantiationStrategy::PoolingCopyOnWrite,
```

Comment:
curious why this was changed?

Reply:
The "Interpreted" option is super slow and afaik not used anymore. Here in the test service it was not important which one to use, but for benchmarks I switched in to "compiled", which should also be fine for all other use-cases of this crate.

Reply:
The interpreted execution method is going to be removed *very* soon anyway: https://github.com/paritytech/substrate/pull/13800